### DISCUSSION TITLE
Temporary error upon first time TOTP registration

### DISCUSSION THREAD

**Discussion by ghost** [neos-sdi/adfsmfa#295] [General]
Hello.

During the TOTP registration flow i encountered an error that clears itself up after clicking sign in with other options. After that it continues working but i do not understand why this happens.

![スクリーンショット 2023-04-17 午後11 11 11](https://user-images.githubusercontent.com/96893300/232687270-3764e9bd-6a6c-4b15-82e2-703676f35187.png)
![スクリーンショット 2023-04-17 午後11 11 20](https://user-images.githubusercontent.com/96893300/232687276-85f1f577-1a18-41d9-a24f-b533142a1dc4.png)

**Comment by redhook62** [comment]
Hi @lambda-makes-things 

Can you send us more logs?
In the windows eventlog when the error occurs. 
In 
- Applications 
- ADFS Log.

regards

**Comment by ghost** [comment]
![スクリーンショット 2023-04-18 午後5 02 30](https://user-images.githubusercontent.com/96893300/232930231-3a41b3d2-50a4-4aae-bca8-168e6f52f5f8.png)
im pretty sure this is an event that relates to this, not the same in the screenshot though

**Comment by redhook62** [comment]
This is a generic (uncatched) exception, and provides little detail. are there any others ?

Have you correctly filled in all the properties concerning the config ? such as the name of the company (Issuer)
Apparently you use SAML, do you provide the UPN claim ?
You use a Mac, do you have the same problems with Windows and Linux ?

regards

**Comment by ghost** [comment]
@redhook62 

Yes, Yes, and Yes other tenant users have this same predicament

**Comment by redhook62** [comment]
Hi @lambda-makes-things 

If you don't have more logs available. simplest way and send me your configuration.

In PowerShell: **Export-MFASystemConfiguration -ExportFilePath c:\path\config.xml**
Then replace confidential information within the file with placeholders.

Send me this file (zip) to the email address indicated in the source codes

regards

**Comment by redhook62** [comment]
News ?
